setup.py

```diff
@@ -160,4 +173,4 @@ def get_version():
           'Bug Reports': 'https://github.com/rism-digital/verovio/issues',
           'Source': 'https://github.com/rism-digital/verovio',
       },
-)
+      )
```

Comment:
I think this indentation level is not correct? It goes with `setup`, doesn't it?

Reply:
This comes from autopep8, so I guess it's correct.